README.md

```diff
@@ -32,7 +32,7 @@ Wiring
 ------
 
 WS2812 driver is using SPI bus. Connect your LED's input wire to the SPI bus 0
-MOSI (pin GP16 on Wipy) Connect LED's power and ground wires to VIN and GND.
+MOSI (pin GP16 on Wipy, pin P22 by default on LoPy <=> pin G9 on extension board) Connect LED's power and ground wires to VIN and GND.
```

Comment:
Could you please do a list, it will be easier to add other boards :
```
WS2812 driver is using SPI bus. Connect your LED's input wire to the SPI bus 0, MOSI is
 * Wipy : pin GP16
 * LoPy : pin P22 by default (G9 on extension board)

Connect LED's power and ground wires to VIN and GND.
```